clang/test/CIR/CodeGen/string-literals.c

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-cir %s -o %t.cir
+// RUN: FileCheck --check-prefix=CIR --input-file=%t.cir %s
+// RUN1: %clang_cc1 -triple aarch64-none-linux-android21 -fclangir -emit-llvm %s -o %t.ll
```

Comment:
Yea, that's the idea, thanks. One more thing: `RUN1` -> should be `RUN` to actually run.

Reply:
That's my bad. After the fix I found that ClangIR and Vanilla LLVM have the following diff in the generated IR:
```
 // LLVM-LABEL: @literals
-// LLVM:  global %struct.anon {
+// LLVM:  global %struct.anon.1 {
 // LLVM:    [10 x i8] c"1\00\00\00\00\00\00\00\00\00",
 // LLVM:    [10 x i8] zeroinitializer,
 // LLVM:    [10 x i8] zeroinitializer
-// LLVM:  }, align 1
+// LLVM:  }
```
The first changed line seems to be OK (just naming) but the second one (align) might be important.

Note: The diff was not introduced by my change and its something else